src/miniAudicle.pro

```diff
@@ -23,7 +23,7 @@ OBJECTS_DIR = build
 
 PRECOMPILED_HEADER = qt/miniAudicle_pc.h
 
-LIBS += -lqscintilla2
+CONFIG += qscintilla2

```

Comment:
I know using `CONFIG` here works, but why do you prefer it over `LIBS`?

Reply:
It wouldn't compile without changing this line (I guess `-lqscintilla2_qt5` might work too).

`CONFIG` is what is mentioned in [QScintilla's documentation](https://www.riverbankcomputing.com/static/Docs/QScintilla/).